samples/snippets/src/main/java/com/example/spanner/CreateBackupWithEncryptionKey.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+// [START spanner_create_backup_with_encryption_key]
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.Timestamp;
+import com.google.cloud.spanner.Backup;
+import com.google.cloud.spanner.BackupId;
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerException;
+import com.google.cloud.spanner.SpannerExceptionFactory;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.encryption.CustomerManagedEncryption;
+import com.google.cloud.spanner.encryption.EncryptionConfigs;
+import com.google.spanner.admin.database.v1.CreateBackupMetadata;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import org.threeten.bp.LocalDateTime;
+import org.threeten.bp.OffsetDateTime;
+
+public class CreateBackupWithEncryptionKey {
+
+  static void createBackupWithEncryptionKey() throws InterruptedException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project";
+    String instanceId = "my-instance";
+    String databaseId = "my-database";
+    String backupId = "my-backup";
+    String kmsKeyName = "my-key";
+
+    try (Spanner spanner =
+        SpannerOptions.newBuilder().setProjectId(projectId).build().getService()) {
+      DatabaseAdminClient adminClient = spanner.getDatabaseAdminClient();
+      createBackupWithEncryptionKey(
+          adminClient,
+          projectId,
+          instanceId,
+          databaseId,
+          backupId,
+          kmsKeyName);
+    }
+  }
+
+  static Void createBackupWithEncryptionKey(DatabaseAdminClient adminClient,
```

Comment:
Is there a specific reason that this method is returning `Void` instead of `void`?

Reply:
Yes, because `Callable` requires a return, so it is easier to the sample like this:

```
    String out = SampleRunner.runSample(() ->
        CreateDatabaseWithEncryptionKey.createDatabaseWithEncryptionKey(
            databaseAdminClient,
            projectId,
            instanceId,
            databaseId,
            key
        ));
```

Than like this:

```
    String out = SampleRunner.runSample(() -> {
        CreateDatabaseWithEncryptionKey.createDatabaseWithEncryptionKey(
            databaseAdminClient,
            projectId,
            instanceId,
            databaseId,
            key
        );
       return null;
    });
```

Personally I don't think it matters much on `void` vs `Void`, but if you have strong opinions I can change it.

Reply:
Cool, I didn't notice that one. Seems like a good reason.